### PR TITLE
UIU-1257: Focus should be on pane header title when Loans list is open

### PR DESCRIPTION
**Pull request by Javokhir12**
## Purpose
- Currently when you access the Open/Closed Loans list, the initial focus is outside the page.
- Requirement: initial focus should be on pane header title.
- Refs: https://issues.folio.org/browse/UIU-1257

## Approach
- Pass `paneTitleRef` prop to `Pane` component to move focus on pane header on `Mount`

### CODE REVIEW

File: src/views/LoansListing/LoansListing.js
Comment (zburke) [inline src/views/LoansListing/LoansListing.js:32]: Don't disable lint. The reformatting here is minor; please just do what it asks :) You can rearrange things here, or move the `createRef` line into a constructor. The acctual cDM methods looks good, of course.

**Review by zburke — CHANGES_REQUESTED** [review]

**Review by zburke — APPROVED** [review]
LGTM! Go ahead and "Squash and merge" unless you want to wait for another approval. Thank you!

**Review by mkuklis — APPROVED** [review]

**Comment by mariia-aloshyna** [comment]
Hi @Javokhir12 Can this PR be merged?

**Comment by Javokhir12** [comment]
> Hi @Javokhir12 Can this PR be merged?

Hi @mariia-aloshyna . Yes we can

**Pull request merged**
